CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * add support for credit cancelling (#385)
 * record retirement locations of ecocredit (#328)
 * add dates as top level fields in credit batches (#393)
+* add project location as field in credit batches (#394)
```

Comment:
Shall we have a sub-section (###) saying that all this added things are for x/ecocredit?